tests/views/test_marriageproperties.py

```
@@ -140,23 +246,20 @@ def test_married_becomes_enabled_after_delete_married_event(scene, noEvents, mp)
-    mp.itemProp("separatedBox", "enabled") == True
-    mp.itemProp("divorcedBox", "enabled") == True
+    assert mp.itemProp("marriedBox", "enabled") == True
+    assert mp.itemProp("separatedBox", "enabled") == True
+    assert mp.itemProp("divorcedBox", "enabled") == True
 
     event = Event(parent=marriage, description="Something happened")
     scene.addItem(event)
-    mp.itemProp("marriedBox", "enabled") == False
-    mp.itemProp("separatedBox", "enabled") == False
-    mp.itemProp("divorcedBox", "enabled") == False
+    assert mp.itemProp("marriedBox", "enabled") == False
```

Ruff (0.13.3)

Avoid equality comparisons to `False`; use `not mp.itemProp("marriedBox", "enabled"):` for false checks

Replace with `not mp.itemProp("marriedBox", "enabled")`

(E712)